sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ParDoLifecycleTest.java

```diff
@@ -307,12 +336,19 @@ DelayedCallStateTracker update(CallState val) {
         latch.countDown();
       }
 
+      int order =
+          callStateVisited.compute(
+              val, (k, v) -> MoreObjects.firstNonNull(v, callStateVisited.size()));
```

Comment:
Variable is unused, perhaps add a comment here that we are computing the order here.